packages/brick-container/src/styles/antd.less

```
@@ -30,3 +30,4 @@
 @import "~@next-core/custom-antd-styles/src/popover.less";
 @import "~@next-core/custom-antd-styles/src/modal.less";
 @import "~@next-core/custom-antd-styles/src/date-picker.less";
+@import "~@next-core/custom-antd-styles/src/slide.less";
```

如果是 `slider`，文件名也用 `slider` 吧？
```suggestion
@import "~@next-core/custom-antd-styles/src/slider.less";
```

---

哦 漏了 `r`